Makefile

```diff
@@ -382,9 +382,14 @@ verify-modules: modules  ## Verify go modules are up to date
 	hack/verify-go-modules.sh
 
 .PHONY: clean
-clean:
-	! test -e $(TOOLS_DIR) || rm -r $(TOOLS_DIR)
-	! test -e $(GOBIN_DIR)/* || rm $(GOBIN_DIR)/*
+clean: clean-workdir
+	! test -d $(TOOLS_DIR) || rm -r $(TOOLS_DIR)
```

Comment:
Super nit-y but would it be simpler to do `rm -rf $(TOOLS_DIR)` instead? And same for GOBIN_DIR and WORK_DIR/.kcp*?

Reply:
Yeah I thought the same, but @sttts did it this way in https://github.com/kcp-dev/kcp/pull/2562 so I decided to stick with the current approach - I agree that would be simpler and I can't see how the test/rm will be any safer?

Reply:
Cool, I'd vote to just do `rm -rf`